tests/post/steps/test_volume.py

```diff
@@ -26,3 +134,152 @@ def check_storage_class(host, name, k8s_apiclient):
         if isinstance(exc, ApiException) and exc.status == 404:
             assert False, 'StorageClass {} not found'.format(name)
         raise
+
+
+@then(parsers.parse("the Volume '{name}' is '{status}'"))
+def check_volume_status(host, name, status, k8s_custom_client):
+    def _check_status():
```

Comment:
Define only once `_check_status`

Reply:
The scope of the function is local so, I think this should not be a problem.
 By the way, the check status function is specific in each case it has been used. Maybe we should really give this nested functions proper names

Reply:
>  Maybe we should really give this nested functions proper names

We should (and we started to do so with `_check_status` and `_check_size`), it's easier for grepping & jumping in the code.